.github/workflows/conda-win.yml

```diff
@@ -0,0 +1,70 @@
+name: Windows Conda End-to-end Test
```

Comment:
Please use the same file

Reply:
I think Windows needs `powershell` and the other file uses `micromamba shell`. So for now I am using separate file. After resolving the issues locally I will combine the two files together.